### PR TITLE
Add-ons warning is raised twice for multi table synthesizers. 

### LINKED ISSUE #1768: Add-ons warning is raised twice for multi table synthesizers.
### Environment Details

* SDV version: 1.9.0

### Error Description

As a user, when I define a column relationship for which I don't have access to the material to use it properly, I see the same warning twice when defining a multi-table synthesizer. The `warning` is:

`UserWarning: The metadata contains a column relationship of type 'address'. which requires the address add-on. This relationship will be ignored. For higher quality data in this relationship, please inquire about the SDV Enterprise tier.`

### Expected behavior

The warning should be raised just once.

### Steps to reproduce

```python
from sdv.multi_table import HMASynthesizer
from sdv.datasets.demo import download_demo

mt_data, mt_metadata = download_demo('multi_table', 'fake_hotels')
mt_metadata.add_column('hotels', 'lat', sdtype='latitude')
mt_metadata.add_column('hotels', 'lon', sdtype='longitude')

mt_metadata.add_column_relationship('hotels', 'gps', ['lat', 'lon'])
mt_synth = HMASynthesizer(mt_metadata)
```

### PR DESCRIPTION
**Pull request by R-Palazzo**
CU-86az7mwk1
Resolve #1768

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-86az7mwk1 SDV - Add-ons warning is raised twice for multi table synthesizers.  #1768](https://app.clickup.com/t/86az7mwk1)

**Review by lajohn4747 — APPROVED** [review]
LGTM

File: sdv/multi_table/base.py [hunk 1]
```diff
@@ -71,7 +71,10 @@ def _print(self, text='', **kwargs):
 
     def __init__(self, metadata, locales=None, synthesizer_kwargs=None):
         self.metadata = metadata
-        self.metadata.validate()
+        with warnings.catch_warnings():
+            warnings.filterwarnings('ignore', message=r'.*column relationship.*')
+            self.metadata.validate()
```
Comment (amontanez24) [inline sdv/multi_table/base.py:76]: where is the other warning being raised?

Comment (R-Palazzo) [reply sdv/multi_table/base.py:76]: It's raised here:
https://github.com/sdv-dev/SDV/blob/21b064a75f06f6f68997110b7e2260fb73026341/sdv/multi_table/base.py#L93

When we defines the single table synthesizers:
https://github.com/sdv-dev/SDV/blob/21b064a75f06f6f68997110b7e2260fb73026341/sdv/multi_table/base.py#L55

Comment (amontanez24) [reply sdv/multi_table/base.py:76]: Got it. Maybe in the future we should have a way to skip the validation if it has already been called from the multi-table level

**Review by amontanez24 — APPROVED** [review]

**Pull request merged**